go/libkb/logout.go

```diff
@@ -53,6 +53,13 @@ func (mctx MetaContext) LogoutUsernameWithOptions(username NormalizedUsername, o
 		return err
 	}
 
+	if options.KeepSecrets {
+		err = g.Env.GetConfigWriter().SetStayLoggedOut(true)
```

Comment:
I think the reason TestBotSignup is failing is that we're calling this too often (when logout gets called by login, e.g.). Maybe we should call it from the engine instead? WDYT @patrickxb ?